### ISSUE TITLE
Draw out the User-Flow Chart

### ISSUE THREAD

**Issue by cml2377** [cml2377/Coronacation#8]
GREAT JORB MAGS!!!!

**Comment by magsmagsmags** [comment]
This is version 1. It includes basic page-to-page navigation and login/signup process. 

![flow 1](https://user-images.githubusercontent.com/57575667/76987653-89792580-6911-11ea-9b4e-a439928d7055.png)

**Comment by magsmagsmags** [comment]
This is version 2. Its updates include an initial map of all basic functionality for the application. It conveys the Features and flow, but it  is hard to follow and can be organized better in the next version. 

![flow2](https://user-images.githubusercontent.com/57575667/76987847-d8bf5600-6911-11ea-9df5-3684858b0459.png)

**Comment by magsmagsmags** [comment]
[Version 3]. It contains the same information as v2, but it was made to be more user-friendly and readable. v3 is awaiting peer review and input. These updates will be reflected in the next version. 

![flow3](https://user-images.githubusercontent.com/57575667/76989171-efff4300-6913-11ea-9f58-bf5433f11b7f.png)

@cml2377

**Comment by N0matic** [comment]
These are really great Mags!!!!

**Comment by N0matic** [comment]
Is there a template you're following? It would be nice if the 'page' sections were a little larger to distinguish them a little more for clarity.

**Comment by magsmagsmags** [comment]
![flow4](https://user-images.githubusercontent.com/57575667/77102294-96b41400-69e6-11ea-9be4-fe3cf3bde023.png)

**Comment by magsmagsmags** [comment]
v5
![flow5](https://user-images.githubusercontent.com/57575667/77105973-6a9b9180-69ec-11ea-87bf-180c40994403.png)

**Comment by magsmagsmags** [comment]
![flow6](https://user-images.githubusercontent.com/57575667/77108419-290ce580-69f0-11ea-852b-3b6fa3297476.png)


v6. flow has flow now.

**Comment by magsmagsmags** [comment]
[Version 7] v7 in the finalized draft for Group Review. 
![flow7](https://user-images.githubusercontent.com/57575667/77109626-3a56f180-69f2-11ea-8496-03dc6f3cdd28.png)
